server/cluster/cluster.go

```diff
@@ -591,6 +592,27 @@ func (c *RaftCluster) HandleStoreHeartbeat(stats *pdpb.StoreStats) error {
 	return nil
 }
 
+// processBucketHeartbeat update the bucket information.
+func (c *RaftCluster) processBucketHeartbeat(buckets *metapb.Buckets) error {
+	c.RLock()
+	region := c.core.GetRegion(buckets.GetRegionId())
+	c.RUnlock()
+	if region == nil {
+		return errors.Errorf("region %v not found", buckets.GetRegionId())
+	}
+	for i := 0; i < retry; i++ {
+		old := region.GetBuckets()
```

Comment:
It breaks the comment:
```
// RegionInfo records detail region info.
// Read-Only once created.
type RegionInfo struct {
	term              uint64
```
Is there any other place to modify it directly?

Reply:
And after the members of the region change with the epoch increase, new `regionInfo` will be created and replace the old one, is the bucket missing?

Reply:
And after the members of the region change with the epoch increase, new `regionInfo` will be created and replace the old one, is the bucket missing?

Reply:
if the confVer of the region has changed,  the bucket of the new leader should be same, so  the origin region should inherit the buckets in the processor of region heartbeat deatils in region.Inherit(origin). so the bucket of RegionInfo should not miss.